### ISSUE TITLE
Keyboard controls and custom alt/title texts

### ISSUE THREAD

**Issue by Cjschmidt** [serebrov/emoji-mart-vue#83]
We are currently using this package on websites which needs to be WCAG2.1 compliant. In order to be compliant, you need to be able to control everything with keyboard-only.

Is it possible to add keyboard control to this package? Right now you can't select or cycle trough the emojis using keyboard only.

Also is it possible to add custom titles/alt texts to the emojis?

**Comment by serebrov** [comment]
@Cjschmidt I think both keyboard control and titles / alt texts are possible.

It would be good to start with the specification: how exactly the keyboard control would work? which key combinations should be used? how it would behave if there are several emoji components on the page?

Same for titles / alt texts - how would the configuration look like? do you mean to specify custom titles for all emojis (there are many of them) or just override the defatuls?

**Comment by mathieutu** [comment]
Hi.
I also would like to add keyboard navigation in picker. 

Here you have both example from native mac picker, and slack one (which is the most perfect picker I've seen): 

![2021-02-04 10 43 52](https://user-images.githubusercontent.com/11351322/106874513-ef130380-66d5-11eb-820c-a36b79545769.gif)

![2021-02-04 10 46 33](https://user-images.githubusercontent.com/11351322/106874905-65176a80-66d6-11eb-8349-12f3d66aba96.gif)

**Comment by serebrov** [comment]
Related issue: #104 (with a suggested reference change from the original React emoji-mart library), I can start with that and see if it would also solve this request.

**Comment by serebrov** [comment]
I've just published version 10.0.0 with keyboard controls and accessibility improvements (use `button` tags for clickable emojis, add `aria-` properties).

Keyboard controls are activated when the cursor is in the search field - emoji can be selected with arrow keys and confirmed with Enter (the `select` event is emmited by picker, same as when the emoji is clicked with mouse).
